sale_medical_prescription/demo/sale_order_demo.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record id="sale_order_medical_order_1" model="sale.order">
+        <field name="pharmacy_id" ref="medical_pharmacy.medical_pharmacy_1" />
+        <field name="partner_id" ref="portal.demo_user0_res_partner" />
```

Comment:
[Travis Build Fail](https://travis-ci.org/OCA/vertical-medical/jobs/187223678#L1196)
`"External ID not found in the system: portal.demo_user0_res_partner" while parsing /home/travis/build/OCA/vertical-medical/sale_medical_prescription/demo/sale_order_demo.xml:7`